etl/src/energuide/cli.py

```diff
@@ -23,7 +23,15 @@ def main() -> None:
               envvar=database.EnvVariables.collection.value,
               default=database.EnvDefaults.collection.value)
 @click.option('--filename', type=click.Path(exists=True), required=True)
-def load(username: str, password: str, host: str, port: int, db_name: str, collection: str, filename: str) -> None:
+@click.option('-a', '--append', is_flag=True, help='Sets load to append instead overwrite')
```

Comment:
Defaults to `False`?

Reply:
Can we update the `help` statement to something like `Append data instead of overwriting`

Reply:
It's a flag, so it'd be called like `energuide load --append --filename filename.zip`, not `energuide load --append True --filename filename.zip`. It's about if it is present or not, so lack of argument means `False`.